src/picasso-definition/interactions/tap-mini-chart/get-tap-position.js

```diff
@@ -8,9 +8,9 @@ export default function getTapPosition({ e, chart }) {
 
```

Comment:
There is not really any reason to prefix files / modules like these with `get`, can simoly be named `tap-position.js`.

Reply:
got it!